Templates/Java/requests_extensions/BaseEntityCollectionReferenceRequest.java.tt

```diff
@@ -6,6 +6,12 @@
 <#=writer.WriteHeader()#>
 <#=host.CreatePackageDef()#>
 
+import <#=host.CurrentModel.GetNamespace().AddPrefix()#>.options.QueryOption;
```

Comment:
perf, global comment: when we have `host.CurrentModel.GetNamespace().AddPrefix()` more than once in a single file, can we replace that with a local variable instantiated once per file?

Reply:
fixed